app/views/questions/index.html.erb

```diff
@@ -21,3 +21,4 @@
        <% end %>
      </tbody>
    </table>
+   <div class="center"><%= paginate @questions %></div>
```

Comment:
Good job figuring out how to add this easily. It should help out load time immensely!